src/ExpressiveInstaller/Resources/config/container-auryn.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+use Auryn\Injector;
+use Northwoods\Container\InjectorContainer;
+use Psr\Container\ContainerInterface;
+
+// Load configuration
+$config = require __DIR__ . '/config.php';
+
+// Create injector
+$injector = new Injector();
+
+// Configuration will be used globally
+$injector->share('config')->delegate('config', function () use ($config) {
+    // Auryn requires that all injections resolve to an object.
+    // Wrap the config array with an ArrayObject to appease the gods.
+    return new \ArrayObject($config);
+});
+
+// Container will be shared globally
+$injector->share(ContainerInterface::class);
+
+// Container will resolve by injector
+$injector->alias(ContainerInterface::class, InjectorContainer::class);
+
+// Container will be created from injector
+$injector->define(InjectorContainer::class, [':injector' => $injector]);
+
+// Inject services
+if (! empty($config['dependencies']['services'])) {
```

Comment:
A small utility in `Northwoods\Container` (something like `(new LoadServiceManagerAlikeConfiguration)($container, $config)` could reduce this code by a lot, and would allow testing the produced injector properly